### ISSUE TITLE
Vault Remove command doesn't updates the folders in code-workspace file

### ISSUE THREAD

**Issue by Harshita-mindfire** [dendronhq/dendron#3648]
## Please select the area your bug applies to. (Multiple selections are Possible. You can leave blank if you're not sure.)

- [x] Workspace.  VSCode workspace, vaults, Intellisense/autocomplete, Dendron settings
- [ ] Lookup.  Dendron's Lookup Command
- [ ] Views.  Dendron Preview, Tree View, Side Panels in the UI
- [ ] Schema.  Dendron Schemas
- [ ] Pod.  Data import from / export to Dendron
- [ ] Publish.  External Site Publish
- [ ] Markdown.  Markdown features, such as syntax support and features

## Describe the bug

`Vault Remove command does not remove the folder from code workspace file.

## To Reproduce

1. In a multi vault workspace, run Vault Remove command
2. see vault is removed from dendron.yml, but not from code workspace.

## Expected behavior
vault folder should be removed from both code workspace and dendron.yml

## Desktop (please complete the following information)

- Browser (if with published websites; include version): [e.g. chrome, safari]
- Versions (all that apply to bug):
  - Dendron Extension: 0.114
  - VS Code version: 1.71
  - Dendron CLI:

## Dendron Log file

Please attach the output of `> Dendron:Dev: Open Logs` here

**Comment by hikchoi** [comment]
>  This can be a windows only issue

Yup. I can't reproduce this on a mac. It indeed might be a windows issue.

**Comment by hikchoi** [comment]
@Harshita-mindfire since you already assigned yourself: [[Vault Remove Command Doesnt Updates the Folders in Code Workspace File|dendron://private/task.2022.10.11.vault-remove-command-doesnt-updates-the-folders-in-code-workspace-file]]
